flight-icons/README.md

```diff
@@ -0,0 +1,35 @@
+# flight-icons
+
+An addon for the icons from `flight`. Currently in development. This addon can be used in React or Ember apps.
+
+## Installation
+
+```bash
+yarn add @hashicorp/flight-icons
+```
+
+## Usage
+
+We intend for icons to be used at a 16 or 24 `viewBox` size.
```

Comment:
suggestion: maybe we can make the text more clear

Maybe I am missing something, but from what I understand here we're saying/assuming that the only way to consume the icons is via <SVG><use/></SVG>? otherwise the word "viewBox" used in this way can be misleading (it is/was for me, when I read it the first time). Could we use a more generic word, like container box, or parent container, or something similar?

In this context ("Usage") maybe we could be even just say:
_We intend for icons to be used at a 16px or 24px size_
(this is what @dizzyup intended, from what I understand) and then explain the sizing logic (below) in a more general way, about the sizes as expressed in the design/UI and which of the two variants (16 vs 24) use depending on the case.